application/apps/indexer/parsers/src/someip.rs

```diff
@@ -0,0 +1,330 @@
+use crate::{Error, LogMessage, Parser};
+use std::{fmt, fmt::Display, io::Write};
+
+use someip_messages::*;
+use someip_payload::fibex::FibexModel;
+use someip_payload::fibex2som::FibexTypes;
+use someip_payload::som::{SOMParser, SOMType};
+
+use log::debug;
+use serde::Serialize;
+
+pub struct SomeipParser<'m> {
+    model: &'m FibexModel,
```

Comment:
we need to be able to create a someip-parser even without a FibexModel. In this case we will only parse the header and display the payload as bytes.
So the datatype for `model` should be an `Option`

Reply:
done.